collect_app/src/androidTest/java/org/odk/collect/android/support/FormActivityTestRule.java

```diff
@@ -10,28 +10,41 @@
 import org.junit.runner.Description;
 import org.junit.runners.model.Statement;
 import org.odk.collect.android.activities.FormEntryActivity;
+import org.odk.collect.android.activities.SplashScreenActivity;
 import org.odk.collect.android.injection.DaggerUtils;
 import org.odk.collect.android.external.FormsContract;
 import org.odk.collect.android.storage.StorageSubdirectory;
+import org.odk.collect.android.support.pages.FirstLaunchPage;
 import org.odk.collect.android.support.pages.FormEntryPage;
 import org.odk.collect.forms.Form;
 
+import java.util.List;
+
 public class FormActivityTestRule implements TestRule {
```

Comment:
Please convert to kotlin as you are here.

Reply:
Converted.